components/trezor_bridge/BUILD.gn

```diff
@@ -0,0 +1,12 @@
+static_library("trezor_bridge") {
+  sources = [
+    "trezor_bridge_handler.cc",
+    "trezor_bridge_handler.h",
+  ]
+  deps = [
+    "//base",
+    "//brave/components/resources:static_resources",
+    "//brave/components/resources:strings_grit",
+    "//content/public/browser",
+  ]
+}
```

Comment:
You would add the grit target for trezor_bridge_resources to this BUILD.gn

Reply:
done